MARIE_SEQ2SEQ/data_generation/README.md

```diff
@@ -0,0 +1,122 @@
+# Data generation for Marie
+
+## Installation
+
+### Prequisites
+- `python3`
+- `python3-venv`
+
+### Setup steps
+
+1. Create a virtual environment and install required dependencies by executing `./install_script_pip.sh -v -i -e` in a bash shell. 
+1. Activate the virtual environment.
+
+   `(Windows)`
+
+   ```cmd
+   $ data_generation_venv\Scripts\activate.bat
+   (data_generation_venv) $
+   ```
+
+   `(Linux)`
+
+   ```cmd
+   $ source data_generation_venv\bin\activate
+   (data_generation_venv) $
+   ```
+
+## Usage
+
+### Dataset creation
+
+Example usage
+
+```python
+import json
+from datetime import datetime
+
+from data_generation.create_training_data_from_kg import DatasetFromKgMaker
+
+
+kg_endpoint = "http://localhost:9999/blazegraph/namespace/kg/sparql"
+example_maker = DatasetFromKgMaker(kg_endpoint=kg_endpoint)
+examples = example_maker.make_examples()
+
+with open(f'train.json', "w") as f:
+    json.dump(examples, f, indent=4)
+```
```

Comment:
This code suggests that training data will be generated from something in the KG. Could you add a few sentences on what kinds of things should be in the KG? That would be highly useful...

Reply:
I've added an elaboration in the paragraph that precedes this code block.